lib/server.js

```diff
@@ -28,21 +45,76 @@ app.get('/people/:id', (req, res, next) => {
   res.json(record[0]);
 });
 
+/**
+ * @route POST /api/v1/:model
+ * Model must be a proper model, located within the ../models folder
+ * @param {model} model.path.required
+ * @returns {object} 200 - Count of results with an array of results
+ * @returns {Error}  500 - Unexpected error
+ */
+
 // Route to Create a person
 app.post('/people', (req, res, next) => {
   let record = req.body;
-  // what can you do to ensure there are no
-  // duplicate ids?
-  record.id = Math.random();
+  record.id = db.teams.length+=100;
```

Comment:
interesting approach to get unique ids - however it would cause issues if you delete records: 

```
db: [1, 2, 3, 4]
db.length: 4
new id = 4 + 100 = 104
new db: [1, 2, 3, 4, 104]
```

after a deletion: 
```
db = [1, 2, 4, 104]
db.length = 4
new id = 4 + 100
new db: [1, 2, 4, 104, 104]
```

Also, you're using the `+=` operator, which is designed to "add and then assign". But you can't assign a new value to `db.teams.length`, so you should really just be using `+`

You were probably meaning to use `db.people.length` here as well, my guess this is a copy paste error